scripts/openwebui.py

```diff
@@ -0,0 +1,52 @@
+# /// script
+# requires-python = ">=3.11,<3.12"
+# dependencies = [
+#     "open-webui==0.6.0",
+# ]
+# ///
+
+import socket
+from contextlib import closing
+import os
+import base64
+import uvicorn
+import random
+from pathlib import Path
+
+KEY_FILE = Path.cwd() / ".webui_secret_key"
+
+os.environ["FROM_INIT_PY"] = "true"
+os.environ['WEBUI_SECRET_KEY'] = "l*cals3cre7*"
```

Comment:
What does this get us? You can fully turn off auth with `WEBUI_AUTH="False"` in the environment